test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -93,6 +102,41 @@ static public void testCreatePoint2D() throws Throwable {
 		assertEquals(getY.invoke(point2D), yNew);
 	}
 
+	/*
+	 * Test array of Point2D
+	 */
+	@Test(priority=2)
+	static public void testCreateArrayPoint2D() throws Throwable {
+		int x1 = 0xFFEEFFEE;
+		int y1 = 0xAABBAABB;
+		int x1New = 0x11223344;
```

Comment:
the x1new ... y2new are not needed

Reply:
@tajila  Would I also need xNew...y4New? OR is it safe to remove it?

Reply:
you can remove them